lib/capture_go_web/views/games_view.ex

```diff
@@ -9,6 +9,18 @@ defmodule CaptureGoWeb.GamesView do
 
   alias CaptureGoWeb.LiveGame
   alias CaptureGo.Goban
+  alias CaptureGo.Accounts.User
+  alias CaptureGo.Games.Game
+
+  def participant?(game, user)
```

Comment:
this heading is currently unnecessary

Reply:
https://github.com/Giesch/capture_go/pull/1/commits/a79da40aa37515bd1541f2bb538ad88230f587f3